person.rb

```diff
@@ -2,7 +2,7 @@
 
 class Person < Nameable
```

Comment:
- [ ] Your class is missing the other method for adding a rental. Kindly implement it following the earlier suggestion in the Book class above.